src/__tests__/node_redis.spec.js

```diff
@@ -133,7 +196,13 @@ tape('NodeJS Redis', function (t) {
 
         client.once(client.Event.SDK_READY_TIMED_OUT, assert.fail);
 
+        const start = Date.now();
         client.once(client.Event.SDK_READY, async () => { // Use SDK_READY event.
+          // ready promise is resolved
```

Comment:
Can't we do these promise checks in parallel instead of sequentially? After all, that and the event are siblings.